### PR TITLE
Add template for Ads

### PR DESCRIPTION
**Pull request by totop716**

### CODE REVIEW

**Review by RitzyDevBox — COMMENTED** [review]
Looks good.  We should consider refactoring the logic for the lists to be more generic so we don't have to replicate the reducers and hooks each time.

File: src/components/FarmCard/FarmCard.tsx [hunk 1]
```diff
@@ -196,6 +200,22 @@ const FarmCard: React.FC<{
...
         />
       )}
+      {stakingInfo.sponsored && stakingInfo.sponsorLink && (
+        <Box className='farmSponsoredLink'>
+          <Trans
+            i18nKey='learnmoreproject'
+            components={{
+              alink: (
+                <a
+                  href='https://quickswap-layer2.medium.com/you-voted-for-a-1-1000-token-split-to-make-quick-more-appealing-9c25c2a2dd7e'
```
Comment (sameepsi) [inline src/components/FarmCard/FarmCard.tsx:210]: Why are we using hard coded link? It should be stakingInfo.sponsorLink

Comment (sameepsi) [reply src/components/FarmCard/FarmCard.tsx:210]: @totop716

File: src/components/SyrupCard/SyrupCard.tsx [hunk 2]
```diff
@@ -98,6 +101,22 @@ const SyrupCard: React.FC<{ syrup: SyrupInfo; dQUICKAPY: string }> = ({
...
         <SyrupCardDetails syrup={syrup} dQUICKAPY={dQUICKAPY} />
       )}
+      {syrup.sponsored && syrup.sponsorLink && (
+        <Box className='syrupSponsoredLink'>
+          <Trans
+            i18nKey='learnmoreproject'
+            components={{
+              alink: (
+                <a
+                  href='https://quickswap-layer2.medium.com/you-voted-for-a-1-1000-token-split-to-make-quick-more-appealing-9c25c2a2dd7e'
```
Comment (sameepsi) [inline src/components/SyrupCard/SyrupCard.tsx:111]: same here

**Review by sameepsi — APPROVED** [review]

**Pull request merged**